sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -158,11 +172,20 @@ export interface HttpHeaders extends Iterable<[string, string]> {
 // @public
 export type HttpMethods = "GET" | "PUT" | "POST" | "DELETE" | "PATCH" | "HEAD" | "OPTIONS" | "TRACE";
 
+// @public
+export interface InMemoryBlob {
+    content: Uint8Array;
+    type?: string;
+}
+
 // @public
 export interface InternalPipelineOptions extends PipelineOptions {
     loggingOptions?: LogPolicyOptions;
 }
 
+// @public
+export function isMultipartRequestBody(body: RequestBodyType | undefined): body is MultipartRequestBody;
```

Comment:
I wonder why is this exported?

Reply:
It's used by the recorder policy. I don't think there's any harm in exporting it